code/addons/test/src/manager.tsx

```diff
@@ -91,50 +81,102 @@ addons.register(ADDON_ID, (api) => {
       type: Addon_TypesEnum.experimental_TEST_PROVIDER,
       runnable: true,
       watchable: true,
-
       name: 'Component tests',
-      title: ({ crashed, failed }) =>
-        crashed || failed ? 'Component tests failed' : 'Component tests',
-      description: ({ failed, running, watching, progress, crashed, error }) => {
-        const [isModalOpen, setIsModalOpen] = useState(false);
 
-        const errorMessage = error?.message;
+      contextMenu: ({ context, state }, { ListItem }) => {
```

Comment:
1. handle stories without the `'test'` tag.
2. handle when clicking Roots/Dirs/Components, where no leaf stories have the `'test'`, and not show the menu